airbyte-integrations/connectors/destination-dev-null/src/test-integration/kotlin/io/airbyte/integrations/destination/dev_null/DevNullBasicFunctionalityIntegrationTest.kt

```diff
@@ -41,6 +40,4 @@ class DevNullBasicFunctionalityIntegrationTest :
     override fun testMidSyncCheckpointingStreamState() {
         super.testMidSyncCheckpointingStreamState()
     }
-
-    @Test @Disabled("File transfer is not supported") override fun testBasicWriteFile() {}
```

Comment:
this is already controlled via the `supportFileTransfer` flag, so the explicit disabling was redundant.